lib/helpers.bash

```diff
@@ -227,6 +230,21 @@ _bash-it-migrate() {
   fi
 }
 
+_bash-it-version() {
+  _about 'shows current Bash-it version'
+  _group 'lib'
+
+  cd "${BASH_IT}" || return
+
+  echo "Current git SHA:"
+  echo "$(git log --pretty=format:'%h on %aI' -n 1)"
+
+  CURRENT_GIT_SHA=$(git rev-parse --verify HEAD)
```

Comment:
It might be quicker to take the result of the previous `git log` command and then just take the first part of that (e.g. with a Bash Regex or `sed`), instead of forking another `git` command.

I haven't measured any timing, but I would assume that getting the first part of the variable would be quicker than running another `git` command...

Reply:
True - I was using them differently - one was a shortlog, the other a long commit SHA. Since github links  and git log work with a short hash just as well, I can make that change.